apps/mobile/src/features/send/send-form/hooks/use-send-form-btc.tsx

```diff
@@ -1,24 +1,76 @@
+import { useAccountUtxosByPaymentType } from '@/queries/balance/bitcoin-balance.query';
+import { useBitcoinAccounts } from '@/store/keychains/bitcoin/bitcoin-keychains.read';
+import { useSettings } from '@/store/settings/settings';
+import { bytesToHex } from '@noble/hashes/utils';
+import * as btc from '@scure/btc-signer';
+
+import { getBtcSignerLibNetworkConfigByMode, payerToBip32Derivation } from '@leather.io/bitcoin';
+import { SATS_IN_BTC } from '@leather.io/constants';
+import { useAverageBitcoinFeeRates } from '@leather.io/query';
+
 import {
   CreateCurrentSendRoute,
   useSendSheetNavigation,
   useSendSheetRoute,
 } from '../../send-form.utils';
-import { SendFormStxSchema } from '../schemas/send-form-stx.schema';
+import { SendFormBtcSchema } from '../schemas/send-form-btc.schema';
 
 type CurrentRoute = CreateCurrentSendRoute<'send-form-btc'>;
 
 export function useSendFormBtc() {
   const route = useSendSheetRoute<CurrentRoute>();
   const navigation = useSendSheetNavigation<CurrentRoute>();
+  const account = route.params.account;
+  const bitcoinKeychain = useBitcoinAccounts().accountIndexByPaymentType(
+    account.fingerprint,
+    account.accountIndex
+  );
+  const { utxos } = useAccountUtxosByPaymentType({
+    fingerprint: account.fingerprint,
+    accountIndex: account.accountIndex,
+    paymentType: 'p2wpkh',
+  });
+  const { networkPreference } = useSettings();
+  const scureNetworkConfig = getBtcSignerLibNetworkConfigByMode(
+    networkPreference.chain.bitcoin.mode
+  );
+  const { data: feeRates } = useAverageBitcoinFeeRates();
 
+  const sortedUtxos = utxos.sort((utxo1, utxo2) => Number(utxo2.value) - Number(utxo1.value));
   return {
     onGoBack() {
       navigation.navigate('send-select-asset', { account: route.params.account });
     },
     // Temporary logs until we can hook up to approver flow
-    async onInitSendTransfer(values: SendFormStxSchema) {
-      // eslint-disable-next-line no-console, lingui/no-unlocalized-strings
-      console.log('Send form data:', values);
+    async onInitSendTransfer(values: SendFormBtcSchema) {
+      if (!sortedUtxos[0]) throw new Error('no utxo');
```

Comment:
Was this all just for testing? Please don't merge over my work in latest btc send form PR to gen the unsigned psbt.